pkg/pillar/types/newlogtypes.go

```diff
@@ -8,8 +8,12 @@ import (
 )
 
 const (
-	// DevPrefix - device log file prefix string
+	// DevPrefix - general file prefix string for device log files
 	DevPrefix = "dev.log."
+	// DevPrefixUpload - file prefix string for device log files to upload
+	DevPrefixUpload = "dev.log.upload."
+	// DevPrefixKeep - file prefix string for device log files to keep on device
+	DevPrefixKeep = "dev.log.keep."
```

Comment:
So, are we going to change the directories naming scheme?

Reply:
no the directories themselves, but the files inside of them